gdpr/models.py

```diff
@@ -25,7 +34,10 @@ def create_consent(self, purpose_slug, source_object, issued_at=None, tag=None,
         Returns:
             Legal Reason: LegalReason object
         """
-        purpose = purposes_map[purpose_slug]
+        try:
+            purpose = purpose_register[purpose_slug]
+        except KeyError:
+            raise KeyError('Purpose with slug {} does not exits'.format(purpose_slug))
         issued_at = issued_at or timezone.now()
 
         legal_reason, created = LegalReason.objects.get_or_create(
```

Comment:
Dotaz: pokud se mi obnoví legal reason, který byl už expirovaný, kdo deanonymizuje data?

Reply:
Nemělo by se tady nějak volat to `renew`?

Reply:
to bych asi nedovoloval